src/schema/sale/index.js

```diff
@@ -171,90 +155,29 @@ export const SaleType = new GraphQLObjectType({
         resolve: ({ image_versions, image_url }) =>
           Image.resolve({ image_versions, image_url }),
       },
-      currency: {
-        type: GraphQLString,
-      },
-      description: {
-        type: GraphQLString,
-      },
+      currency: { type: GraphQLString },
+      description: { type: GraphQLString },
       display_timely_at: {
         type: GraphQLString,
-        resolve: sale => {
-          const { live_start_at, registration_ends_at, start_at, end_at } = sale
-
-          // Closed
-          if (end_at && end_at < moment()) {
-            return "Ended"
-
-            // End Registration
-          } else if (registration_ends_at > moment()) {
-            const diff = moment().diff(moment(registration_ends_at), "hours")
-            const format = diff > -24 ? "ha" : "MMM D, ha"
-            const label = `Register by\n${moment(registration_ends_at).format(
-              format
-            )}`
-            return label
-
-            // Live Auction
-          } else if (live_start_at) {
-            if (isLiveOpen(sale)) {
-              return "In Progress"
-
-              // Live auction starting soon
-            } else if (live_start_at) {
-              return `Live ${moment(live_start_at).fromNow()}`
-            }
-
-            // Timed Auction
-          } else if (start_at) {
-            const range = moment().add(5, "days")
-            const startAt = moment(start_at)
-            const isInProgress = startAt < moment()
-            const isUpcoming = startAt > moment() && startAt < range
-            const isNearFuture = startAt > range
-            const dateLabel = saleDate => {
-              return (
-                moment(saleDate)
-                  .fromNow()
-                  .replace("in ", "")
-                  .replace("ago", "")
-                  .trim() + " left" // e.g., X min left
-              )
-            }
-
-            // Timed auction in progress
-            if (isInProgress) {
-              return dateLabel(end_at)
-
-              // Coming soon
-            } else if (isUpcoming) {
-              return dateLabel(start_at)
-
-              // Coming in the future
-            } else if (isNearFuture) {
-              return `Ends on ${moment(end_at).format("MMM D, ha")}`
-            }
-          } else {
-            return null
-          }
+        args: {
+          useUpdatedFormatting: { type: GraphQLBoolean, defaultValue: false },
+        },
+        resolve: (
+          sale,
+          { useUpdatedFormatting },
+          _request,
+          { rootValue: { meBiddersLoader } }
+        ) => {
+          return displayTimelyAt(sale, useUpdatedFormatting, meBiddersLoader)
```

Comment:
Might want to swap order of `useUpdatedFormatting` with `meBiddersLoader` since the latter is a hard unchanging dependeny and the the former is transitive... but i guess this doesn't matter if the function accepts an object instead of a series of arguments